README.Rmd

```diff
@@ -1,66 +1,68 @@
 ---
-  output: github_document
+output: github_document
 ---
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-<br>
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
 
 # mrgda <a href='https:/metrumresearchgroup.github.io/mrgda'><img src='man/figures/logo.png' align="right" width="135px"/></a>
 
+<!-- badges: start -->
+[![Build Status](https://github-drone.metrumrg.com/api/badges/metrumresearchgroup/mrgda/status.svg)](https://github-drone.metrumrg.com/metrumresearchgroup/mrgda)
+[![codecov](https://codecov.io/gh/metrumresearchgroup/mrgda/branch/main/graph/badge.svg)](https://codecov.io/gh/metrumresearchgroup/mrgda)
+<!-- badges: end -->
+
 ## Overview 
 
-mrgda is a data assembly helper, providing a set of functions that help 
+`mrgda` is a data assembly helper, providing a set of functions that help 
 you assemble, explore and validate your data set.
 
-An example data assembly script highlighting many `mrgda` functions can be found
-[here](https://ghe.metrumrg.com/templates/msproject/blob/master/script/data-assembly.Rmd).
 
+### Featured Vignettes
+
+* [Getting Started](https://metrumresearchgroup.github.io/mrgda/articles/getting-started.html) -- Basic examples and introduction to a typical data assembly workflow.
```

Comment:
Replace `Basic examples and introduction to a typical data assembly workflow` with something along the lines of `introduction to commonly used data assembly mrgda functions`